lib/ruby_lsp/ruby_lsp_rails/indexing_enhancement.rb

```diff
@@ -3,13 +3,17 @@
 
 module RubyLsp
   module Rails
-    class IndexingEnhancement
+    class IndexingEnhancement < RubyIndexer::Enhancement
       extend T::Sig
-      include RubyIndexer::Enhancement
+
+      sig { params(index: RubyIndexer::Index).void }
+      def initialize(index)
+        super
+        @index = index
+      end
```

Comment:
We don't need to override initialize. This is already the same in the [parent class](https://github.com/Shopify/ruby-lsp/blob/4c7659be56ab83dae65c3bf63e666b95b684f046/lib/ruby_indexer/lib/ruby_indexer/enhancement.rb#L12).